spec/Vehicle/Vehicle.vspec

```diff
@@ -281,6 +277,7 @@
   type: attribute
   unit: s
   description: The time needed to accelerate the vehicle from a given start velocity to a given target velocity.
+  deprecation: V2.1 removed as ambiguous definition (start/stop-speed not defined)
```

Comment:
Anyone using this attribute?

If it is supposed to be an attribute (with a constant value) then it needs to be better defined, e.g. by stating that start speed i 0 km/h and stop speed is 100 km/h. But the question is if an attribute like that is needed? And if so - is an attribute sufficient or would rather an RPC be needed (giving start and stop speed as parameters and getting time needed as return value)

My suggestion is to deprecate it so that we later can remove it. Better to have a limited set of well-defined datapoints than a big set of ambiguous ones.

Reply:
I would vote to remove it (if keeping it, datatype also need to be float)

Reply:
Remove it right away - or first mark it as deprecated? In this case it seems to be ill-defined and not used, so is it OK to remove it instantly, or should we always go via deprecation?